### DISCUSSION TITLE
velero.io not accessible .

### DISCUSSION THREAD

**Discussion by shirishbhalekar** [vmware-tanzu/velero#4601] [Community support Q&A]
Any one know how to access Velero Documentation.

**Comment by Kaffeetasse** [comment]
Use https://velero.netlify.app/

**Reply by theabmitra** [reply]
What is the issue with the older URL - https://velero.io/docs/. Am able to use the one you suggested

**Reply by Kaffeetasse** [reply]
The issue is #4579, strange that you are able to access it.
